index.html

```diff
@@ -196,7 +211,7 @@ <h3>About <br> Myself</h3>
     <div class="contact-form">
```

Comment:
[REQUIRED] You have still not implemented the suggestion of the last two reviewers. If you look at the place I marked with a red line. That section should be white, not grey. I think the problem was that they didn't suggest ways to fix it. I think you should have wrapped the form section and the about section before it in a container element and given that container element a white background.

|Your Portfolio|
|-----------|
|![image](https://user-images.githubusercontent.com/62938735/193407259-403fe1d0-9d21-47a0-af4b-1934912689df.png)|

Reply:
i have implemented as much i could, the section is not white due to background color also tech image i have aligned as much as i could without affecting my mobile version